CRM/Report/Form.php

```diff
@@ -361,6 +361,12 @@ class CRM_Report_Form extends CRM_Core_Form {
   public $_orderBy = NULL;
   public $_orderByFields = array();
   public $_orderByArray = array();
+  /**
+   * Array of clauses to group by.
+   *
+   * @var array
+   */
+  protected $_groupByArray = array();
```

Comment:
Just thinking for consistency with the other helper things in that list I also wonder if inherited classes can support work with protected vars or not